.github/workflows/java8_integration_tests_webui.yml

```diff
@@ -49,7 +49,7 @@ jobs:
           mkdir -p ~/.m2
           ALLUXIO_DOCKER_NO_TTY=true \
           ALLUXIO_DOCKER_GIT_CLEAN=true \
-          ALLUXIO_DOCKER_MVN_PROJECT_LIST=dora/tests,webui \
+          ALLUXIO_DOCKER_MVN_PROJECT_LIST=webui \
```

Comment:
setting only webui would only compile webui? i think it needs to be completely removed

Reply:
Deleted.